pkg/dochandler/handler.go

```diff
@@ -102,25 +102,46 @@ func (r *DocumentHandler) ProcessOperation(operation batch.Operation) (document.
 	return nil, nil
 }
 
-// ResolveDocument returns document based on passed in ID or Document
-func (r *DocumentHandler) ResolveDocument(idOrDocument string) (document.Document, error) {
+// ResolveDocument fetches the latest DID Document of a DID. Two forms of string can be passed in the URI:
+//
+// 1. Standard DID format: did:sidetree:<unique-portion>
+//
+// 2. DID with initial-values DID parameter:
+// did:sidetree:<unique-portion>;initial-values=<encoded-original-did-document>
+//
+// Standard resolution is performed if the DID is found to be registered on the blockchain.
+// If the DID Document cannot be found, the encoded DID Document given in the initial-values DID parameter is used
+// to generate and return as the resolved DID Document, in which case the supplied encoded DID Document is subject to
+// the same validation as an original DID Document in a create operation
```

Comment:
What's the plan for the published flag? https://github.com/decentralized-identity/sidetree/blob/master/docs/protocol.md#published-flag

Reply:
I checked the spec, it is not there yet. It will  be revisited in couple of weeks when we review spec for additional changes.